crates/swc_ecma_ast/src/expr.rs

```diff
@@ -147,6 +147,9 @@ pub enum Expr {
     #[tag("TsAsExpression")]
     TsAs(TsAsExpr),
 
+    #[tag("TsExpressionWithTypeArguments")]
+    TsExprWithTypeArgs(TsExprWithTypeArgs),
```

Comment:
Is this required?
Also, it does not seem like a good name.

Reply:
I think it's required. The name is borrowed from TypeScript and shortened. Or, do you have any suggestions?

Reply:
`Expr:: TsExprWithTypeArgs` does not seems like an instantiation expression. How about `TsInstantiation`?

Reply:
LGTM.